examples/resources/astro_api_token/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "astro_api_token" "example" {
```

Comment:
we should add some examples for 
- org tokens with deployment and workspace roles
- workspace token
- workspace token with deployment role
- deployment token with `DEPLOYMENT_ADMIN` role
- deployment token with custom role